README.md

```diff
@@ -112,7 +125,7 @@ The `--path` flag has no effect on the `npx env-enc set-pw` command as the passw
 The Functions and Functions subscription management commands commands can be executed in the following format:
 `npx hardhat command_here --parameter1 parameter_1_value_here --parameter2 parameter_2_value_here`
 
-Example: `npx hardhat functions-read --network mumbai --contract 0x787Fe00416140b37B026f3605c6C72d096110Bb8`
+Example: `npx hardhat functions-read --network polygonMumbai --contract 0x787Fe00416140b37B026f3605c6C72d096110Bb8`
```

Comment:
I am a little concerned about causing confusion due to changing the network name.  Do you think the verbose name is worth this potential confusion?  If so, you will need to sync with @aelmanaa so he can update the docs accordingly.

Reply:
I have already created a ticket to update docs: https://smartcontract-it.atlassian.net/browse/FUN-368
Besides this "renaming", we will need to include the new network details in the public docs.

Reply:
Great! The rest looks good to me other than the missing block explorer, however if you prefer you can address that separately.  I am going to approve this now.